scylla-gdb.py

```diff
@@ -3644,7 +3644,7 @@ def invoke(self, arg, for_tty):
             initial_task_ptr = int(gdb.parse_and_eval(args.task))
             this_task = self._probe_pointer(initial_task_ptr, args.scanned_region_size, using_seastar_allocator, args.verbose)
             if this_task is None:
-                gdb.write("Provided pointer 0x{:016x} is not an object managed by seastar or not a task pointer\n".format(ptr))
+                gdb.write("Provided pointer 0x{:016x} is not an object managed by seastar or not a task pointer\n".format(initial_task_ptr))
```

Comment:
nit: if this fix is unrelated to your changes, either submit it in a separate PR
or move it to the front of the series so it would be obvious it is applicable before your changes.